apps/main/src/dao/entities/user-gauge-vote-next-time.ts

```diff
@@ -1,39 +1,33 @@
 import { enforce, group, test } from 'vest'
 import { chainValidationGroup, curvejsValidationGroup } from '@/dao/entities/validation/curvejs-validation'
-import type { ChainId } from '@/dao/types/dao.types'
+import type { ChainId, CurveApi } from '@/dao/types/dao.types'
 import { requireLib } from '@ui-kit/features/connect-wallet'
-import type { ChainParams, ChainQuery } from '@ui-kit/lib/model/query'
+import type { ChainQuery, UserQuery } from '@ui-kit/lib/model/query'
 import { queryFactory } from '@ui-kit/lib/model/query'
-import { createValidationSuite } from '@ui-kit/lib/validation'
-import { CurveApi } from '@ui-kit/shared/useApiStore'
+import { userValidationGroup } from '@ui-kit/lib/model/query/user-validation'
+import { createValidationSuite, type FieldsOf } from '@ui-kit/lib/validation'
 
-export const userGaugeVoteNextTimeValidationGroup = ({ enabled }: { enabled: boolean }) =>
-  group('userGaugeVoteNextTimeEnabled', () => {
-    test('enabled', () => {
-      enforce(enabled).message('Enabled should be true')
-    })
-  })
-
-type QueryParams = ChainQuery<ChainId> & { gaugeAddress: string; userAddress: string; enabled: boolean }
-type QueryKeyParams = ChainParams<ChainId> & { gaugeAddress: string; userAddress: string; enabled: boolean }
-
-const _fetchUserGaugeVoteNextTime = ({ gaugeAddress, enabled }: QueryParams) =>
-  requireLib<CurveApi>().dao.voteForGaugeNextTime(gaugeAddress)
+type UserGaugeQuery = ChainQuery<ChainId> & UserQuery<string> & { gaugeAddress: string }
+type UserGaugeParams = FieldsOf<UserGaugeQuery>
 
 export const { useQuery: useUserGaugeVoteNextTimeQuery, invalidate: invalidateUserGaugeVoteNextTimeQuery } =
   queryFactory({
-    queryKey: (params: QueryKeyParams) =>
+    queryKey: (params: UserGaugeParams) =>
       [
         'user-gauge-vote-next-time',
         { chainId: params.chainId },
         { gaugeAddress: params.gaugeAddress },
         { userAddress: params.userAddress },
-        { enabled: params.enabled },
```

Comment:
@OnlyJousting it is not necessary to add `enabled` here. Every `useQuery` hook allows `enabled` as 2nd parameter.
However, in this case we should just validate `gaugeAddress` I believe